evalai/utils/challenges.py

```diff
@@ -0,0 +1,84 @@
+import os
+import requests
+import sys
+
+from click import echo
+
+from pylsy import pylsytable
+
+from evalai.utils.auth import get_headers
+from evalai.utils.urls import Urls
+from evalai.utils.common import Text, valid_token
+
+
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+
+
+def print_challenges_tables(challenge):
+    challenge_title = Text.title.format(challenge["title"])
+    challenge_id = "ID: " + Text.idfield.format(challenge["id"])
+
+    title = "{} {}".format(challenge_title, challenge_id)
+
+    description = "{}\n".format(challenge["short_description"])
+    date = "End Date : " + challenge["end_date"].split("T")[0]
+    date = Text.subtitle.format(date)
+    challenge = "{}{}{}{}".format(title, description, date, Text.br)
+    return challenge
+
+
+def get_challenges(url):
+
+    headers = get_headers()
+    try:
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        echo(err)
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(err)
+        sys.exit(1)
+
+    response_json = response.json()
+    if valid_token(response_json):
+
+        challenges = response_json["results"]
+        if len(challenges) is not 0:
+            for challenge in challenges:
+                challenge = print_challenges_tables(challenge)
```

Comment:
There is discrepancy here. The function says `challenges_tables` but you only print a challenge.
So you can either move the for loop into the function or rename the function to `print_challenge_table`.